.pre-commit-config.yaml

```diff
@@ -51,7 +47,7 @@ repos:
       - id: mypy
         args:
           - --ignore-missing-imports
-          - --install-types  # See mirrors-mypy README.md
+          - --install-types # See mirrors-mypy README.md
```

Comment:
I like two space much more. It visually separates the code from the comments

Reply:
That's done by the formatter: https://github.com/redhat-developer/yaml-language-server